audius-cli

```diff
@@ -878,6 +883,30 @@ def auto_upgrade(ctx, remove, cron_expression):
             job.setall(cron_expression)
 
 
+@cli.command()
+@click.pass_context
+def launch_pedalboard(ctx):
```

Comment:
maybe we add a network check here and if it equals prod, don't deploy?

Reply:
can also add a flag in the app itself that returns errors if it's configured for prod but still stands up